R/read_extract_homelessness.R

```diff
@@ -3,9 +3,13 @@
 #' @inherit read_extract_acute
 #'
 #' @export
-read_extract_homelessness <- function(
-    year,
-    file_path = get_boxi_extract_path(year = year, type = "Homelessness")) {
+read_extract_homelessness <- function(year,
+                                      file_path = get_boxi_extract_path(year = year, type = "Homelessness")) {
+  # Specify years available for running
+  if (year < 1617) {
```

Comment:
```suggestion
  if (year < "1617") {
```

Reply:
The string will apply to Delayed discharges only where this is supplied by the team and not via BOXI extract